runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -217,4 +231,25 @@ private void lookupPlayer(String playerName)
 			hiscorePanel.lookup(playerName);
 		});
 	}
+
+	private HiscoreEndpoint getLocalHiscoreEndpointType()
+	{
+		EnumSet<WorldType> worldType = client.getWorldType();
+		if (worldType.contains(WorldType.LEAGUE))
+		{
+			return HiscoreEndpoint.LEAGUE;
+		}
```

Comment:
This part of the logic could be replaced by `HiscorePanel#selectWorldEndpoint()`, e.g.:

```java
private HiscoreEndpoint getLocalHiscoreEndpoint()
{
	final HiscoreEndpoint profile = HiscorePanel.selectWorldEndpoint();
	if (profile != HiscoreEndpoint.NORMAL)
	{
		return profile;
	}

	switch (client.getAccountType())
	// ...
}
```